crates/libs/windows/src/core/guid.rs

```diff
@@ -98,6 +103,24 @@ impl core::convert::From<&str> for GUID {
     }
 }
 
+impl core::convert::From<u128> for GUID {
+    fn from(value: u128) -> Self {
+        Self::from_u128(value)
+    }
+}
+
+impl core::convert::From<GUID> for u128 {
+    fn from(value: GUID) -> Self {
+        value.to_u128()
+    }
+}
+
+impl core::convert::From<&GUID> for u128 {
```

Comment:
We don't impl `From<&u128> for GUID` so I'm not sure we need to do the reverse. `GUID` is also `Copy` so it's trivial to go from a `&GUID` to `GUID`.

Reply:
Ok, removed From<&GUID>